docs/en/flash.md

```diff
@@ -15,13 +15,33 @@ Supported platforms: OS X, Linux, Windows, anything that runs Python
 
 Run the following command to flash an *aggregated* binary as is produced for example by the [cloud build service](build.md#cloud-build-service) or the [Docker image](build.md#docker-image).
 
-`esptool.py --port <USB-port-with-ESP8266> write_flash -fm <mode> -fs <size> 0x00000 <nodemcu-firmware>.bin`
+`esptool.py --port <serial-port-of-ESP8266> write_flash -fm <mode> -fs <size> 0x00000 <nodemcu-firmware>.bin`
 
 - `mode` is `qio` for 512&nbsp;kByte modules and `dio` for 4&nbsp;MByte modules (`qio` might work as well, YMMV).
 - `size` is given in bits. Specify `4m` for 512&nbsp;kByte and `32m` for 4&nbsp;MByte.
 
 Check the [esptool flash modes documentation](https://github.com/themadinventor/esptool#flash-modes) for details and other options.
 
+## esptool-ck
+> Esptool reads the compiled program in ELF format, extracts code and data sections, and either dumps a section to a file or assembles the firmware file from several segments. Esptool also communicates with the bootloader to upload firmware files to flash. Esptool can automatically put the board into UART bootloader mode using a variety of methods.
+
+Source: [https://github.com/igrr/esptool-ck](https://github.com/igrr/esptool-ck)
+
+Supported platforms: Windows, OS X, Linux, or any other POSIX compatible OS with a C compiler. If you're a Windows or Linux user, you can also [download the binary release here](https://github.com/igrr/esptool-ck/releases).
+
+Run the following command to flash an combined binary image.
```

Comment:
Please use the same wording *aggregated* like above to avoid confusion about differences.